### DISCUSSION TITLE
Gerar boleto de pagamento ingressos CERTI

### DISCUSSION THREAD

**Discussion by PySulBR** [apyb/comunidade#131] [Geral]
Olá,
A CERTI pediu para nós gerarmos um boleto para o pagamento dos ingressos da Python Sul 2024.
A data de vencimento deve ser 02/09.
O valor é R$ 1120,00.
**Razão Social:** Fundação Centros de Referência em tecnologias inovadoras
**CNPJ:** 78.626.363/0001-24

Agradeço desde já,
Talita Rossari, co-organizadora da Python Sul 2024.

**Comment by cecivieira** [comment]
[boleto_com_recibo_00036670390000000001.pdf](https://github.com/user-attachments/files/16786823/boleto_com_recibo_00036670390000000001.pdf)

@PySulBR segue o boleto solicitado

**Reply by PySulBR** [reply]
uhuuul, aprendemos a gerar boletossss (embora tão anos 2010 isso) \o/

**Reply by cecivieira** [reply]
hahahahahahahahah
Num é???
Foi até bem fácil. A gente só não tinha usado essa funcionalidade antes, pois a habilitamos há pouquíssimo tempo.
